Annotato/Annotato/ViewModels/Annotation/AnnotationViewModel.swift

```diff
@@ -274,3 +306,12 @@ extension AnnotationViewModel {
         enterMinimizedMode()
     }
 }
+
+// MARK: WebSocket Actions
+extension AnnotationViewModel {
+    func updateAnnotation() {
```

Comment:
```suggestion
    func broadcastAnnotationUpdate() {
```

Will this convey the intent better?